GPU/Common/FramebufferCommon.cpp

```diff
@@ -1934,3 +1934,45 @@ bool FramebufferManagerCommon::GetOutputFramebuffer(GPUDebugBuffer &buffer) {
 	return true;
 }
 
+// This function takes an already correctly-sized framebuffer and packs it into RAM.
+// Does not need to account for scaling.
+// Color conversion is currently done on CPU but should be done on GPU.
```

Comment:
(Except using the GPU might cause problems because of various implementations' dithering behavior and games that expect exact colors like Danganronpa, so we can't entirely be rid of the CPU path.)

-[Unknown]